cpu/riscv_common/Makefile.dep

```diff
@@ -0,0 +1,10 @@
+# Tell the build system that the CPU depends on the risc-v common files:
+USEMODULE += riscv_common
+
+# include common periph code
+USEMODULE += riscv_periph_common
+
+USEMODULE += periph_pm
```

Comment:
I think this line is not needed because the periph_pm feature is always added if provided by the platform:

https://github.com/RIOT-OS/RIOT/blob/38188017a810105b67a3771735287c34d3fdba42/Makefile.dep#L89-L09

 And the FEATURES_REQUIRED is turned into a USEMODULE:

https://github.com/RIOT-OS/RIOT/blob/38188017a810105b67a3771735287c34d3fdba42/makefiles/features_modules.inc.mk#L7-L10

So you can drop this line:

```suggestion
```

Reply:
Thanks, removed!